jck/jck-semiauto-updater/jckupdater.sh

```diff
@@ -167,50 +171,126 @@ isLatestUpdate() {
 	curl -o "build.txt" $GIT_URL -H "Authorization: token $GIT_TOKEN"
 	echo -e "JCK version in build.txt:\n$(cat build.txt)\n\n"
 	
-	if grep -q "$JCK_WITHOUT_BACKSLASH" build.txt; then
+	#check if latest exclude file is available
+	getExcludeFiles
+	exclude_files=$?
+
+	result=$(grep -q "$JCK_WITHOUT_BACKSLASH" build.txt; echo $?)
+
+	if [[ $result -eq 0 &&  $exclude_files -eq 0 ]]; then
 		echo " JCK$JCK_VERSION material is $JCK_WITHOUT_BACKSLASH in the repo $GIT_URL. It is up to date. No need to pull changes"
 		cleanup
 		exit 2
-	else	
-		echo " JCK$JCK_VERSION $JCK_WITHOUT_BACKSLASH is latest and not in the repo $GIT_URL... Please proceed with download"
-		get_JAVA_SDK
+	else
+		if [ $result -ne 0 ]; then
+			echo " JCK$JCK_VERSION $JCK_WITHOUT_BACKSLASH is latest and not in the repo $GIT_URL... Please proceed with download"
+			getJavaSDK
+		else
+			echo " Latest exclude files are available for JCK$JCK_VERSION"
+		fi
 		getJCKSources
 	fi
 }
 
+getLastModifiedForFile() {
+    file="$1"
+    curl -s -H "X-JFrog-Art-Api:$ARTIFACTORY_TOKEN" "$ARTIFACTORY_API_URL$file" | awk -F'"' '/lastModified/ {print $4}'
+}
+
+getExcludeFiles(){
+	github_last_modified=$(curl -s -H "Authorization: token $GIT_TOKEN" "https://api.github.ibm.com/repos/runtimes/JCK$JCK_VERSION-unzipped/commits?path=excludes/jck$JCK_FOLDER_SUFFIX.jtx" | awk -F'"date":' '{if(NF>1){gsub(/[",]/,"",$2); print $2; exit}}')
```

Comment:
- Please do not hardcode a specific repo
- It is not limited to one `jck$JCK_FOLDER_SUFFIX.jtx`. There could be other files.

Reply:
In artifactory material we have only .jtx file in `/exc` . So we are comparing last modified of .jtx file from Git repo with artifactory material to decide if update is needed or not.

Reply:
Created a parameter `GIT_EXCLUDE_API_URL` to provide API url in JCK_Sync job.